wirecard/qmore.py

```diff
@@ -74,9 +74,9 @@ def init_datastorage(self, orderIdent,
         return result
 
     def init_frontend(self, amount, currency, paymentType, language,
-                      orderDescription, successUrl, cancelUrl, failureUrl,
-                      serviceUrl, confirmUrl, consumerUserAgent,
-                      consumerIpAddress, autoDeposit=None,
+                      orderDescription, customerStatement, successUrl,
```

Comment:
We use this library other places than for the tickets. Isn't it better if you don't have to supply the customerStatement argument if you don't need to?